scripts/manual_review_analyses.py

```diff
@@ -21,18 +21,58 @@ def get_analysis_data(client, type, id):
-    if type == 'PATIENT':
-        return client.post_patient_analyses_variants_export(id)['exportId']
-    elif type == 'INHERITANCE':
-        return client.post_inheritance_analyses_variants_export(id)['exportId']
-
-
-def get_variants_export(client, type, id, export_id):
-    if type == 'PATIENT':
-        return client.get_patient_analyses_variants_export(id, export_id)
-    elif type == 'INHERITANCE':
-        return client.get_inheritance_analyses_variants_export(id, export_id)
+def post_variants_export(client, analysis_type, variant_type, analysis_id):
+    if analysis_type == 'PATIENT':
+        if variant_type == 'molecular_variant':
+            return client.post_patient_analyses_variants_export(analysis_id)['exportId']
+        elif variant_type == 'copy_number_variation':
+            return client.post_patient_analyses_cnv_export(analysis_id)['exportId']
+    elif analysis_type == 'INHERITANCE':
+        if variant_type == 'molecular_variant':
+            return client.post_inheritance_analyses_variants_export(analysis_id)['exportId']
+        elif variant_type == 'copy_number_variation':
+            return client.post_inheritance_analyses_cnv_export(analysis_id)['exportId']
+
+
+def get_variants_export(client, analysis_type, variant_type, analysis_id, export_id):
+    if analysis_type == 'PATIENT':
+        if variant_type == 'molecular_variant':
+            return client.get_patient_analyses_variants_export(analysis_id, export_id)
+        elif variant_type == 'copy_number_variation':
+            return client.get_patient_analyses_cnv_export(analysis_id, export_id)
+    elif analysis_type == 'INHERITANCE':
+        if variant_type == 'molecular_variant':
+            return client.get_inheritance_analyses_variants_export(analysis_id, export_id)
+        elif variant_type == 'copy_number_variation':
+            return client.get_inheritance_analyses_cnv_export(analysis_id, export_id)
+
+
+def get_variants(client, analysis_type, variant_type, analysis_id, variant_count):
+    export_id = post_variants_export(client, analysis_type, variant_type, analysis_id)
+    export = None
+    while export is None:
+        try:
+            time.sleep(variant_count/100 + 1)  # delay to request exported report, min 1 sec.
+            export = get_variants_export(client, analysis_type, variant_type, analysis_id, export_id)
+        except HTTPError:
+            pass
+
+    return export
+
+
+def count_manual_review_labels(variants_export):
+    manual_review_count = [0, 0, 0]  # manual_review labes Y, Y2, Y3 (rare)
+    for variant in variants_export:
+        variant_labels = variant['classificationTreeLabelsScore']['labels'].lower()  # Correct upercase labels
+
+        if 'y,manual review' in variant_labels:
+            manual_review_count[0] += 1
+        if 'y2,manual review' in variant_labels:
+            manual_review_count[1] += 1
+        if 'y3 (rare),manual review' in variant_labels:
+            manual_review_count[2] += 1
```

Comment:
Alternative, using `count`. If statements are avoided, no match will result in count of 0.
```
manual_review_count[0] += variant_labels.count('y,manual review')
manual_review_count[1] += variant_labels.count('y2,manual review')
manual_review_count[2] += variant_labels.count('y3 (rare),manual review')
```

Is it possible/likely these labels of interest are extended?
In that case, you could consider this code example.
Benefit, only line with declaration of count_labels will change, instead of line 64, + additional if statement. on 74/75
Not certain if `+=` would work on dict values as well.
Main disadventage: three nested for loops (analysis, variants in exports, labels in count_labels)
```
manual_review_count = {}
count_labels = ['y,manual review', 'y2,manual review', 'y3 (rare),manual review']
for label in count_labels:
    manual_review_count[label] = manual_review_count[label] + variant_labels.count(label)
```

Reply:
Nice, implemented the count.